stories/index.js

```diff
@@ -3,8 +3,8 @@ import { storiesOf, action, linkTo } from '@kadira/storybook';
 import { host } from 'storybook-host';
 
 import SnoozePopup from '../src/lib/components/SnoozePopup';
-import MainPanel from '../src/lib/components/MainPanel';
-import ManagePanel from '../src/lib/components/ManagePanel';
+// import MainPanel from '../src/lib/components/MainPanel';
+// import ManagePanel from '../src/lib/components/ManagePanel';
```

Comment:
I flipped a coin. I wasn't sure if you were planning on adding more stories or tests for this, and that's why they were imported initially.

But it's easy enough for me to amend the commit and delete the two lines. Your call.